scripts/jenkins/ardana/gerrit/build_test_package.py

```diff
@@ -445,19 +445,24 @@ def build_test_packages(change_ids, obs_linked_project, home_project,
             try_branches.insert(0, c.target)
         processed_changes.append(c.id)
 
+        # skip packages that don't have asssociated RPMs
+        if c.gerrit_project not in gerrit_project_map():
+            print("Warning: Project %s has no RPM, Skipping" % c.gerrit_project)
+            
         # Create the package if it doesn't exist already
```

Comment:
(nit) this comment is meant for the following line and could go in the else block